Sources/Foundation/NSSwiftRuntime.swift

```diff
@@ -18,6 +18,8 @@
 @_exported import Glibc
 #elseif canImport(Musl)
 @_exported import Musl
+#elseif canImport(Bionic)
+@_exported import Bionic
```

Comment:
Did you try using `import Bionic` in all these other files also? @compnerd asked that we use that instead where possible.

Reply:
Some files can definitely use Bionic instead of Android, so let me see where I can do that.

Reply:
It looks like all lib files right now actually need specifically Android imports, so I don't think there's anything more to simplify here to use Bionic in more places .

Reply:
I tried it myself, turns out there are several that can be modified to `Bionic`, hyp#1.